### PR TITLE
Replace "requests.put/post" in "data_putter.py" with the "perform_request" function from "utils.py"

### PR DESCRIPTION
**Pull request by valyo**
Before submitting a PR to the `dev` branch:
- [ ] Tests passing
- [ ] Black formatting
- [ ] Rebase/merge the `dev` branch
- [ ] Note in the CHANGELOG

Additional checks before submitting a PR to the `master` branch:
- Change version in `dds_cli/version.py` (?)

### CODE REVIEW

File: dds_cli/data_putter.py [hunk 1]
```diff
@@ -399,28 +398,15 @@ def add_file_db(self, file):
...
-
-        return added_to_db, error
+        request_method = "put" if fileinfo["overwrite"] else "post"
+        response_json = dds_cli.utils.perform_request(
+            DDSEndpoint.FILE_NEW,
+            method=request_method,
+            params=params,
+            json=file_info,
+            headers=self.token,
+            error_message="Failed to add file '{file}' to database",
```
Comment (inaod568) [inline dds_cli/data_putter.py:408]:
```suggestion
            error_message=f"Failed to add file '{file}' to database",
```

File: dds_cli/data_putter.py [hunk 2]
```diff
@@ -399,28 +398,15 @@ def add_file_db(self, file):
...
+        )
+
+        added_to_db, message = (True, response_json)
+        return added_to_db, message
```
Comment (inaod568) [inline dds_cli/data_putter.py:412]: In order to make sure that the upload of the rest of the files continue even if one fails we need to add a try except block here, so something like this I believe. 
```suggestion
        try:
            response_json = dds_cli.utils.perform_request(
                DDSEndpoint.FILE_NEW,
                method=request_method,
                params=params,
                json=file_info,
                headers=self.token,
                error_message="Failed to add file '{file}' to database",
            )
            added_to_db, message = (True, response_json)
        except (dds_cli.exceptions.ApiRequestError, dds_cli.exceptions.ApiResponseError, dds_cli.exceptions.DDSCLIException) as err:
            message = str(err)
            LOG.warning(message)

        return added_to_db, message
```

**Review by inaod568 — CHANGES_REQUESTED** [review]

File: dds_cli/data_putter.py (hunk 2 above)
Comment (valyo) [reply dds_cli/data_putter.py]: I pushed the change but not sure how to simulate a file upload failure for testing right now. I continue with other tasks not to lose too much time with this testing

Comment (inaod568) [reply dds_cli/data_putter.py]: Sure, that's fine, I can test and make changes in this if I see any that are needed.

**Review by inaod568 — APPROVED** [review]
This seems to work exactly like intended so 👍🏻

**Pull request merged**